sunpy/time/time.py

```diff
@@ -296,7 +296,7 @@ def _variables_for_parse_time_docstring():
 
 
 @add_common_docstring(**_variables_for_parse_time_docstring())
-def parse_time(time_string, *, format=None, **kwargs):
+def parse_time(time_string : str, *, format:str=None, **kwargs) -> astropy.time.Time:
```

Comment:
I don't think the first type is correct.

Reply:
uhh yes I understood the issue incorrectly, will work on that